### PR TITLE
3gpp Rel16 GTP Messages & IEs

### PR DESCRIPTION
**Pull request by UmakantKulkarni**

### CODE REVIEW

File: scapy/contrib/gtp_v2.py [hunk 1]
```diff
@@ -542,6 +545,56 @@ class IE_BearerFlags(gtp.IE_Base):
                    BitField("PPC", 0, 1)]
 
 
+class IE_MMContext_EPS(gtp.IE_Base):
+    name = "IE MM Context (EPS Security Context and Quadruplets)"
+    fields_desc = [ByteEnumField("ietype", 107, IEType),
+                   FieldLenField("length", None, length_of="length",
+                                 adjust=lambda pkt, x: len(pkt.payload) +
+                                 4, fmt="H"),
+                   BitField("CR_flag", 0, 4),
```
Comment (guedou) [inline scapy/contrib/gtp_v2.py:554]: Could you replace this 4 bits BitField with a ByteField?

Comment (UmakantKulkarni) [reply scapy/contrib/gtp_v2.py:554]: I can; but i was trying to follow the same syntax from other IEs.
Everywhere in this module, we use CR_flag & instance as 4 bit field.

**Comment by guedou** [comment]
This cannot be merged as it is, as it looks that this PR includes commit that should not be there. Please clean your commits history.

**Comment by UmakantKulkarni** [comment]
> This cannot be merged as it is, as it looks that this PR includes commit that should not be there. Please clean your commits history.

Let me open the new PR with the clean commit history.

**Pull request closed without merging**